hudi-client/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -632,6 +647,10 @@ public FileSystemViewStorageConfig getClientSpecifiedViewStorageConfig() {
     return clientSpecifiedViewStorageConfig;
   }
 
+  public boolean getStringFormation() {
+    return Boolean.parseBoolean(props.getProperty("hoodie.tmp.string.format"));
```

Comment:
whats this?

Reply:
It's just for local testing to compare `String.format()` vs `StringBuilder`.  `StringBuilder` is more efficient and the reduced overhead is tangible when the number of records is huge.